tool/site/client-react/src/components/radioField.tsx

```diff
@@ -0,0 +1,31 @@
+import React, {FC} from "react"
+import {Radios} from "nhsuk-react-components"
+import {Field} from "formik"
+
+export interface RadioFieldProps {
+  name: string
+  label: string
+  fieldRadios: Array<Radio>
+  error?: string
+}
+
+interface Radio {
+  value: string
+  text: string
+  defaultChecked?: boolean
+}
+
+const errorStyle: React.CSSProperties = {color: "red", marginBottom: "20px"}
+
+const RadioField: FC<RadioFieldProps> = ({name, label, fieldRadios, error}) => (
+  <Field id={name} name={name} labelProps={{bold: true}} label={label} as={Radios}>
+    {error &&
+        <p style={errorStyle}>{error}</p>
```

Comment:
NHS form fields should have built in error styling.

Also if we do need custom styling:
```
styled.p\`
color: red;
marginBottom: 20px;
\`
```

Reply:
But yeah try just setting the error message on the Radios `<Field>` directly

Reply:
have used the nhs radios react component error prop as above